tests/storage/lvm_test.py

```diff
@@ -1059,6 +1059,18 @@ def test_vg_get_error(fake_devices):
     assert "Fake lvm error" in e.value.value
 
 
+def test_vg_get_invalid_output(monkeypatch, fake_devices):
+    fake_runner = FakeRunner(out=b"Fake lvm output")
+    lc = lvm.LVMCache(fake_runner)
+
+    monkeypatch.setattr(lvm, "_lvminfo", lc)
+
+    with pytest.raises(lvm.InvalidOutputLine) as e:
+        lvm.getVG("vg-name")
```

Comment:
This will raise in this line:
https://github.com/oVirt/vdsm/blob/013692737dc9dd89b977cb8a91e479c8757fd0db/lib/vdsm/storage/lvm.py#L616

Are you sure this is correct?

Reply:
What do you mean? That line was not covered, so I thought that it might be nice to slightly increase code coverage by testing that an invalid output of the lvm command will raise the expected exception. And by doing that, we get rid of the last red line in the test coverage summary for the reload method.

I'd say it is correct.

Reply:
Right, I missed the check for lvm.InvalidOutputLine. We can keep this test, but it should check also how the failure affects the cache - I think the cache is not affected by this since the command did not fail. Not sure if this is good or bad, but this case is not really expected. I would add a TODO: to think if this is the best way to handle this unlikely error.